[2023-12-17T12:17:47.483+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:17:47.487+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:17:47.546+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:17:47.547+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:17:47.612+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-06T04:00:00+00:00 [queued]>
[2023-12-17T12:17:47.633+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-06T04:00:00+00:00 [queued]>
[2023-12-17T12:17:47.633+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 3
[2023-12-17T12:17:47.661+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-06 04:00:00+00:00
[2023-12-17T12:17:47.667+0000] {standard_task_runner.py:57} INFO - Started process 3799 to run task
[2023-12-17T12:17:47.672+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-06T04:00:00+00:00', '--job-id', '551', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpvzzraj8w']
[2023-12-17T12:17:47.674+0000] {standard_task_runner.py:85} INFO - Job 551: Subtask Create_Table
[2023-12-17T12:17:47.765+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:17:47.767+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-06T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:17:47.771+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-06T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T12:17:47.933+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-06T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-06T04:00:00+00:00'
[2023-12-17T12:17:47.954+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T12:17:49.114+0000] {airQuality.py:161} INFO - Table air_quality Already Exist
[2023-12-17T12:17:49.153+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T12:17:49.206+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231206T040000, start_date=20231217T121747, end_date=20231217T121749
[2023-12-17T12:17:49.492+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T12:17:49.748+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:51:50.279+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-06T04:00:00+00:00 [skipped]> on host d67dd4682ccf
[2023-12-28T16:51:50.846+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-06T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-06T04:00:00+00:00'
[2023-12-28T16:51:51.046+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:51:58.270+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:51:58.346+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:51:58.470+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231206T040000, start_date=20231228T165149, end_date=20231228T165158
